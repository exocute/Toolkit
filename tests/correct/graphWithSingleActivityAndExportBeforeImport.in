// forbidden chars

Graph graph
	Export exp2
	Import imp1	
	Activity act1 act1.exe