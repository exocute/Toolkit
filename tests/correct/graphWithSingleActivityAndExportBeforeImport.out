Name: graph
ImportName: imp1
ExportName: exp2
Id: act1, Name: act1.exe
